tensorlayer/rein.py

```diff
@@ -110,7 +124,12 @@ def choice_action_by_probs(probs=[0.5, 0.5], action_list=None):
         The probability distribution of all actions.
     action_list : None or a list of int or others
         A list of action in integer, string or others.
-        - If None, returns an integer range between 0 and len(probs)-1.
+            - If None, returns an integer range between 0 and len(probs)-1.
+
+    Returns
+    --------
+    float/int/str
```

Comment:
Not sure about the correct syntax here. maybe we shall separate them using commas?

Reply:
@lgarithm any idea here?